assembly.xml

```diff
@@ -11,9 +11,6 @@
         <fileSet>
             <directory>src/main/resources/camel</directory>
```

Comment:
This file is outdated sort of, we need to have like 3 of them for the following distributions
- PRP integration
- orders to odoo quotation integration 
- obs based patient to odoo customer integration